### PR TITLE
WMS 11536: Sprints covering OCI O&M

### PR DESCRIPTION
**Pull request by ayoub-belmehdi**
* Init Sprints repo

* Add Sprint 1: Logging Analytics Overview

* Add Sprint 2: Ingest logs in Logging Analytics

* Add Sprint 3: Create a Label

* Add Sprint 4: Create a Parser

* Add Sprint 5: Create a Source

* Add Sprint 6: Upload a Log File and View Logs in Log Explorer

* Add Sprint 7: Create and Trigger an Ingest Time Detection Rule

* Add Sprint 8: Create a Scheduled Search Detection Rule

* Add Sprint 9: Create and Verify an Alarm

* Add Sprint 10: Oracle Database Abnormal Termination

* Add Sprint 11: Sensitive Configuration and Privilege Escalation

* Add Sprint 12: Suspicious Activity and Data Exfiltration

* Add Sprint 13 Container Image Pulled Manually

* Add Sprint 14: DNS Exfiltration on Windows Systems

* Add Sprint 15: Sensitive Data Access from Threat Tagged Location

* Add Sprint 16: Problem Labels for Custom Apps-Logs

* Add Sprint 17: Unsuccessful Food Orders

* Add Sprint 18: Logs from DB Tables

# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by kgvarun** [comment]
WMS ID: 11536

File: oandm/sprint-dns-exfiltration-on-windows-systems/manifest.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "workshoptitle": "LiveLabs Sprints",
+  "help": "livelabs-help-**enterarea**_us@oracle.com",
```
Comment (hope-fisher) [inline oandm/sprint-dns-exfiltration-on-windows-systems/manifest.json:3]: replace with livelabs-help-sprints_us@oracle.com

File: oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+2. Save the log explorer search by clicking on "Actions" and Selecting "Save As..."
+  ![Figure 2: Save the log search](images/save-log-explorer-search.png)
+3. Fill the name, the description and Click on "Save":
+  ![Figure 3: Fill the form to save the search](images/fill-save-search-form.png)
+4. Create a Scheduled Detection Rule by clicking on "Actions" and Selecting "Create detection rule":
+  ![Figure 4: Click on "Create detection rule" button](images/click-on-detection-rule-creation.png)
+5. Fill the Detection Rule form showing up (Refer to [How to create and trigger an ingest time detection rule?](?lab=sprint-create-and-trigger-ingest-time-detection-rule) for more details):
+  ![Figure 5: Fill the form to create the detection rule](images/fill-detection-rule-form.png)
+
+Prepare the logs to upload and trigger the Detection Rule making its metrics available to the alarms service:
```
Comment (hope-fisher) [inline oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md:30]: Use `##` to create a secondary header format

File: oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      </copy>
+
+    ```
+
+    **Explanation:** This script generates dummy logs into a new file & uploads them into your OCI Logging Analytics Service to trigger your Detection Rule.
+
+    Once successfully triggered, the results section of your Detection rule page should show a graph similar to this one:
+    ![Figure 6: Trigger the detection rule](images/wait-for-rule-to-be-triggered.png)
+
+Create the alarm from the Detection Rule:
```
Comment (hope-fisher) [inline oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md:47]: `## ` here as well

File: oandm/sprint-suspicious-activity-and-data-exfiltration/sprint-suspicious-activity-and-data-exfiltration.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+      ![Figure 4: Create a Detection Rule from the Saved Search](images/create-detection-rule-from-saved-search.png)
+
+Your Detection Rule should be up and ready and you should see a notification similar to this one:
+
+![Figure 5: Detection Rule creation Confirmation Message](images/detection-rule-creation-confirmation-message.png)
+
+Trigger the Detection Rule and make its metrics available in the alarms service. The fastest way to achieve this is by signing in to your tenancy with a wrong password.
+
+Create an alarm and verify it:
```
Comment (hope-fisher) [inline oandm/sprint-suspicious-activity-and-data-exfiltration/sprint-suspicious-activity-and-data-exfiltration.md:49]: use `##` to create a secondary header format to differentiate this new set of steps

File: oandm/sprint-sensitive-configuration-and-privilege-escalation/sprint-sensitive-configuration-and-privilege-escalation.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    1. Open the label creation panel:
+    ![Figure 4: Open the label creation panel](images/open-label-creation-panel.png)
+    2. Fill the label fields & Save the label:
+    ![Figure 5: Fill the label information & click on "create"](images/create-label.png)
+    3. Save the conditional Label:
+    ![Figure 6: Check the Conditional Label & Create](images/check-conditional-label-and-create.png)
+
+Your new conditional label should now be ready to be used in other Monitoring services.
+
+Create a Detection Rule:
```
Comment (hope-fisher) [inline oandm/sprint-sensitive-configuration-and-privilege-escalation/sprint-sensitive-configuration-and-privilege-escalation.md:48]: `##` to create secondary header

File: oandm/sprint-problem-labels-for-custom-apps-logs/sprint-problem-labels-for-custom-apps-logs.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+4. Specify a **Name** and **Description (optional)**.
+
+5. Add the following to **Example Base Field Content**.
+      ```Example base field content
+         <copy>Jan 04 21:03:24 XXXX_F5_DMZXX err dcc[11457]: 9999999999:9: [SECEV] Request blocked, violations: Web scraping detected. HTTP protocol compliance sub violations: N/A. Evasion techniques sub violations: N/A. Web services security sub violations: N/A. Virus name: N/A. Support id: 99999999999999999, source ip: 192.168.1.100, xff ip: N/A, source port: 99999, destination ip: 203.128.45.67, destination port: 999, route_domain: 200, HTTP classifier: /Common/www.xxxxxxxxx.yy.http, scheme HTTPS, geographic location: <RU>, request: <GET /ns/xxxxxx.yyy?id_seccion=9999 HTTP/1.1\r\nContent-Length: 0\r\nCookie: XXXXXXXX_XXXXXXXXX=d8b78f937f6f9d569cda500fd5cae49>, username: <8117.1533970714@MAILCATCH.COM>, session_id: <d9ba5ea0f4e98df0></copy>
+      ```
+      ![](./images/example-log-content.png "Example log content")
+
+7. Specify the following regular expression at **Parser regular expression**.
```
Comment (hope-fisher) [inline oandm/sprint-problem-labels-for-custom-apps-logs/sprint-problem-labels-for-custom-apps-logs.md:21]: Numbering skips 6?

File: oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+  **Variable values (These fields do not alter the result of the live lab):**
+
+    - **Rule Name:** Choose a name for your rule, ours will be "Oracle Database Abnormal Termination DR" for this Live Lab.
+    - **Metric Namespace:** Pick a friendly name for your metric namespace to use later when creating your alarm, ours will be "livelabsayoub_ingestmetrics" for this Live Lab.
+    - **Metric Name:** Pick a friendly name for your metric, ours will be "terminations" for this Live Lab.
+
+    ![Figure 2: Fill "Rule name" field](images/fill-dr-variable-fields-and-save.png)
+
+Your Detection Rule should be up and ready and you should see a screen similar to this one:
```
Comment (hope-fisher) [inline oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md:24]: `##` to create secondary header format

File: oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+![Figure 3: Detection Rule creation Confirmation Message](images/detection-rule-creation-confirmation-message.png)
+
+Prepare the logs to upload and trigger the Detection Rule making its metrics available to the alarms service:
```
Comment (hope-fisher) [inline oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md:28]: ## to create secondary header format

File: oandm/sprint-log-explorer-for-analytics-and-visualizations/sprint-log-explorer-for-analytics-and-visualizations.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+4. **Fields panel**, where you can select sources and fields to filter your data.
+
+5. **Visualization panel**, where you can select the way to view search data in a form that helps you.
+
+6. **Main panel**, where the visualization output appears above the results of the query.
+
+## Using Visualizations Panel and Fields Panel
+
+1. **Visualization Panel**:
```
Comment (hope-fisher) [inline oandm/sprint-log-explorer-for-analytics-and-visualizations/sprint-log-explorer-for-analytics-and-visualizations.md:30]: Omit numbering here?

**Review by hope-fisher — CHANGES_REQUESTED** [review]
please see individual comments for feedback. Keep in mind that the `##` are suggestions, but please keep formatting consistent within this Sprint family. Using the secondary heading format when starting a new set of steps is recommended.

File: oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md (hunk 2 above)
Comment (ayoub-belmehdi) [reply oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md:34]: Done!

File: oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md (hunk 3 above)
Comment (ayoub-belmehdi) [reply oandm/sprint-container-image-pulled-manually/sprint-container-image-pulled-manually.md:53]: Done!

File: oandm/sprint-suspicious-activity-and-data-exfiltration/sprint-suspicious-activity-and-data-exfiltration.md (hunk 4 above)
Comment (ayoub-belmehdi) [reply oandm/sprint-suspicious-activity-and-data-exfiltration/sprint-suspicious-activity-and-data-exfiltration.md:53]: Done!

File: oandm/sprint-sensitive-configuration-and-privilege-escalation/sprint-sensitive-configuration-and-privilege-escalation.md (hunk 5 above)
Comment (ayoub-belmehdi) [reply oandm/sprint-sensitive-configuration-and-privilege-escalation/sprint-sensitive-configuration-and-privilege-escalation.md:52]: Done!

File: oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md (hunk 7 above)
Comment (ayoub-belmehdi) [reply oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md:26]: Done!

File: oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md (hunk 8 above)
Comment (ayoub-belmehdi) [reply oandm/sprint-oracle-database-abnormal-termination/sprint-oracle-database-abnormal-termination.md:32]: Done!

**Comment by hope-fisher** [comment]
@ayoub-belmehdi please see the comments above for final edits needed.

**Comment by ayoub-belmehdi** [comment]
> @ayoub-belmehdi please see the comments above for final edits needed.

@hope-fisher We resolved the comments above (The ones above your last comment), Are there any other comments that we are missing please?

**Review by hope-fisher — APPROVED** [review]

**Pull request merged**